backend/lcfs/db/seeders/common/role_seeder.py

```diff
@@ -13,59 +13,71 @@ async def seed_roles(session):
     """
 
     roles_to_seed = [
+        {
+            "name": RoleEnum.GOVERNMENT,
```

Comment:
It's a good idea to add pre-defined record IDs as well. I think I missed including them in the first version of the seeders.